challenges/backend/app.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask, request, abort, jsonify
+import json
+
+# The file in which the numbers get saved to in JSON format.
+# This way if the server happens to go down the data will persist.
+STORED_FILE_NAME = 'numbers.txt'
```

Comment:
`numbers.txt` works here for the file name but a better one would probably start with a `.` so it is hidden and have the `.json` file ending.